CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ x.x.x Release notes (yyyy-mm-dd)
 * Realm Object Server errors not explicitly recognized by the client are now reported to the application
   regardless.
 * Add support for JSON Web Token as a sync credential source.
+* Add support for Nickname Auth  as a sync credential source.
```

Comment:
This should presumably mention anonymous credentials as well. There's also an extra space in the middle of this sentence.